Bug 1137968 - [B2G] Create Ukrainian keyboard.

RudyLu opened the pull request.

ltedone committed:
Bug 1137968 - Create Ukrainian keyboard r=ltedone

lightsofapollo commented:
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bb73a1be8f365eb9f12daca6b71da843125fe24c

RudyLu committed:
Bug 1137968 - Add Ukrainian keyboard layout.
   - Fix the keyboard integration test for this change.
   - Also remove the execute access from some of the layout files.

lightsofapollo commented:
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=78eb8a58c88404266e8d4f32812841b4ce969b6a

lightsofapollo commented:
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b48c8e428dabb489d3ae4cd69f53a8c4dbd1ebbb

lightsofapollo commented:
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=23637d5f22ab2d4f4f560a9397dc78eb6731f0fc
